i18n/keys.pot

```diff
@@ -17,229 +17,163 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java:66
```

Comment:
Can you remove this file from the commit? I'm not sure if it will conflict with Crowdin after the merge (and it's not related).

Reply:
Done